k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -39,7 +39,8 @@ pub enum DeleteResources {
 /// Delete Upgrade resource.
 #[derive(clap::Args, Debug)]
 pub struct DeleteUpgradeArgs {
-    /// If set then upgrade will skip the io-engine pods restart
+   /// Used for fetching the name of the upgrade resources.
+   /// Pass the name of the branch to which upgrade has happened earlier.
```

Comment:
Looks like an indentation issue...
```suggestion
    /// Used for fetching the name of the upgrade resources.
    /// Pass the name of the branch to which upgrade has happened earlier.
```